frameworks/rooch-nursery/sources/multisign_account.move

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) RoochNetwork
+// SPDX-License-Identifier: Apache-2.0
+
+/// Bitcoin multisign account module
+module rooch_nursery::multisign_account{
+
+    use std::vector;
+    use std::option;
+    use moveos_std::signer;
+    use moveos_std::object::{Self, Object};
+    use rooch_framework::address_mapping;
+    use rooch_framework::bitcoin_address::{Self, BitcoinAddress};
+
+
+    const ErrorInvalidThreshold: u64 = 1;
+    const ErrorMultiSignAccountNotFound: u64 = 2;
+    const ErrorInvalidParticipant: u64 = 3;
+    const ErrorParticipantMustHasBitcoinAddress: u64 = 4;
+    const ErrorParticipantAlreadyJoined: u64 = 5;
+    const ErrorInvalidPublicKey: u64 = 6;
+
+    struct MultiSignAccountStore has key {
+    }
+
+    struct MultiSignAccount has key {
+        /// The multisign account address
+        bitcoin_address: BitcoinAddress,
+        /// The multisign account rooch address
+        address: address,
```

Comment:
May it be named `rooch_address`? The syntax shows it was the built-in `address`.